org.moneymanagerex.MMEX.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.moneymanagerex.MMEX
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: mmex
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+modules:
+  - name: wxWidgets
+    config-opts:
+       - --disable-shared
```

Comment:
are all of those options really required? Were they copied from somewhere?

Reply:
They're copied from here:

https://github.com/moneymanagerex/moneymanagerex/tree/master/dockers

Reply:
Maybe try if removing  `--disable-shared` / `--enable-cxx11` / `--with-cxx=11` helps

Reply:
not if the cmakefiles are broken by using improper detection mechanisms.

Reply:
It doesn't appear that removing those flags helps with the build succeeding.

@Erick555 @ovari do you have recommendations?

Reply:
> @ovari do you have recommendations?

Sorry, don't know about the code. Perhaps @n-stein has, or knows someone who has, some ideas? Thank you

Reply:
The build fail in the app because of WebView.

In wxWidget configure output:
```
checking for WEBKIT... configure: WARNING: webkitgtk not found.
configure: WARNING: WebKit not available, disabling wxWebView
```

I believe it's because it is looking for WebKitGtk 4.0 but the runtime has 4.1 (it's a 4.1 API, that break). Last time I looked at building wxWidget with WebKitGtk 4.0 it failed because wxWidget relies on the libsoup2.4 API (4.0) while 4.1 uses libsoup-3.0. And libsoup documentation is seriously lacking on how to migrate so I gave up.

The most recent runtime with 4.0 is GNOME 42 that was deprecated the day 44 was released.

Reply:
Also see https://github.com/flathub/flathub/pull/4184#discussion_r1208587227